### ISSUE TITLE
Training、Validation and Test

### ISSUE THREAD

**Issue by rabbit218218** [LMD0311/PointMamba#50]
First of all, thank you for the update on the work! 
I currently have a question: After executing the command `CUDA_VISIBLE_DEVICES=<GPU> python main.py --finetune_model --config cfgs/finetune_modelnet.yaml --ckpts <path/to/pre-trained/model> --exp_name <name>`, does this directly yield the final result of 93.6% that you recorded on GitHub? Or is that the result after running the test? Because I noticed there is code related to testing, but I didn't see any code for executing the test.

**Comment by LMD0311** [comment]
The code you mentioned is to load the checkpoint and then finetune.

To test a specific checkpoint, you may refer to https://github.com/LMD0311/PointMamba/issues/41.

**Comment by rabbit218218** [comment]
> 您提到的代码是加载检查点，然后进行微调。
> 
> 要测试特定的检查点，您可以参考 [#41](https://github.com/LMD0311/PointMamba/issues/41)。

Alright, thank you for your response. So, does that mean our work did not involve dividing a designated test set for testing? Are the results on the "Code" page the best accuracy (acc) selected after fine-tuning?

**Comment by LMD0311** [comment]
The evaluation after each epoch during the training period is already conducted on the test set. This aligns with other work.

Please refer to the config lines with `subset: 'test'`:
https://github.com/LMD0311/PointMamba/blob/80858e096e33ff7470640fe1f2132e5c1e667074/cfgs/finetune_scan_objbg.yaml#L18-L21
